### ISSUE TITLE
MVD Cloud Deployment Instructions Update

### ISSUE THREAD

**Issue by zeier** [agera-edc/MinimumViableDataspace#222]
Feature agera-edc/DataSpaceConnector#194

## Description

Instructions to [set up cloud deployment](https://github.com/eclipse-dataspaceconnector/MinimumViableDataspace/blob/main/docs/developer/continuous_deployment.md) contain all the relevant information and links to set up a cloud deployment.

The instructions are however hard to follow if someone without Azure experience runs through the steps for the first time. E.g., to set up a service identity for GitHub actions, a link has to be followed which in turn contains a link to the description which explains how to set up the service identity. In situations, when strictly following the instructions provided by the linked articles, the user will have to take unnecessary steps to navigate through the Azure portal.

The goal of this story is to improve the instructions to make them easier to follow.

## Acceptance Criteria

- [ ] Continuous deployment setup instructions define the steps to be taken instead of only providing links
- [ ] Screenshots are available to make the steps easier to follow
- [ ] Usability of instructions are validated with a team-external stakeholder

## Tasks

- [ ] Extend instructions to make them more comprehensible
- [ ] Validate instructions by having a team-external stakeholder setting up a cloud deployment

**Comment by chrislomonico** [comment]
From the latest deployment experience:

I have run through this a few times to ensure that the errors have been addressed. From what IO can see running against my own private azure environment, the issues have been fixed.
A few things to note:

- I run against a fully cleaned azure env with the exception of the app registrations and the enterprise app
- I run the `init` action without issue
- I immediately run the `deploy` action and there is an issue where the ACR is not ready and the container images built in deploy did not all succeed.
- I cleaned the one image that made it to the ACR and ran deploy again and all succeeded.

Based on this behavior I am led to believe that there can be a delay between creation of the ACR and when it is ready for utilization (deployment to the repositories). This can be handled with an update to the documentation to indicate that there should be a slight delay between running the `init` and `deploy` actions.
